blocks/gnav/gnav.js

```diff
@@ -124,6 +124,21 @@ class Gnav {
       menu.querySelector('h2').remove();
       navItem.appendChild(navLink);
 
+      if (navLink.href.match('newsletter-modal.com')) {
```

Comment:
I get it, but not sure I like it 😉  Maybe we could target the subscribe link through an (exotic) formatting of the link, or by picking the 5th or last `h2` on the page?

If we do want to stick with a URL as identifier, I'd suggest using something like `https://blog.adobe.com#subscribe`...

Reply:
I don't know about targeting the last h2, because in the future, if they want to remove the button, it would simply be replaced by the previous one.

I thought about targetting the textContent + location but then if they translate the button, it won't work, so for right now I think it's best we keep to the URL. I will update it to your suggestion!